apps/web/src/components/Feed/Socialize/CommentBox/CommentBox.tsx

```diff
@@ -108,6 +108,16 @@ export function CommentBox({ queryRef, onSubmitComment, isSubmittingComment }: P
 
   const handleInputKeyDown = useCallback<KeyboardEventHandler>(
     async (event) => {
+      /* To disable formatting, manually handling shortcuts might be the most straightforward approach. 
+        While it can be a bit verbose, it ensures that we have full control over what formatting shortcuts 
+        are allowed. 
+      */
+      if (
+        (event.ctrlKey || event.metaKey) && // Ctrl or Cmd
+        ['b', 'i', 'u', 's'].includes(event.key)
+      ) {
+        event.preventDefault(); // Prevent formatting
+      }
```

Comment:
Looks good!

btw is there any default attributes that we can used? 🤔

<img width="840" alt="CleanShot 2023-08-18 at 13 11 28@2x" src="https://github.com/gallery-so/gallery/assets/4480258/9aeaf827-be4e-4fa4-9311-c88ed860b8d5">

Reply:
I didn't fully tested yet, it could be similar to this

```tsx
        <Textarea
          onPaste={handlePaste}
          onKeyDown={handleInputKeyDown}
          ref={textareaRef}
          onInput={handleInput}
          spellCheck={false}
        />
```

If it doesnt work as expected, then we can go to your changes

Reply:
nope, it didn't work. these shortcuts are browser and operating system-specific behaviors, and i think they can be hard to directly control through HTML attributes or properties alone..